api/src/app.py

```diff
@@ -27,22 +28,31 @@
 
 prefix = "/api/v1"
 
+oauth2_scheme = OAuth2AuthorizationCodeBearer(authorizationUrl="", tokenUrl="")
+
+
+def auth_with_jwt(jwt_token: str = Security(oauth2_scheme)):
+    # Authentication is handled by DMSS. Adding a security dependenciy is
+    # necessary to generate correct JSON schema for openapi generator.
+    pass
+
 
 def create_app():
     from features.blueprints import blueprints
     from features.entity import entity
     from features.jobs import jobs
     from features.system import system
 
-    # Using public routes, since authentication is handled by DMSS
     public_routes = APIRouter()
+    authenticated_routes = APIRouter()
     public_routes.include_router(blueprints.router)
     public_routes.include_router(entity.router)
```

Comment:
Should not these routes also have auth?

Reply:
looks like these routes do not use dmss_api